pkg/ingester/transfer.go

```diff
@@ -214,6 +214,13 @@ func (i *Ingester) TransferTSDB(stream client.Ingester_TransferTSDBServer) error
 	fromIngesterID := ""
 
 	xfer := func() error {
+
+		dir, err := ioutil.TempDir("", "tsdb_xfer")
```

Comment:
I would suggest to let configure this as well (ie. `-experimental.tsdb.transfer-temp-dir` / `transfer_temp_dir`). We should also `Validate()` that `TransferTempDir != Dir`.

Reply:
seems like an unnecessary configuration option, `TempDir` ensures that there isn't a naming conflict so it wont have that collision.

Reply:
Ok